hcs-sxc-java-examples/hcs-sxc-java-settlement-demo/config/docker-compose.sample.yml

```diff
@@ -53,6 +67,45 @@ services:
       - 8084:8084
 
   Erica:
+<<<<<<< HEAD
+    container_name: Erica
+    image: hederahashgraph/hcs-sxc-java-settlement-demo:latest
+    restart: on-failure
+    environment:
+      APP_ID: 4
+      PUBKEY: 302a300506032b6570032100bcabbb31c4c6418ea323e02dd46c060f82936141b5d2d2a1da89e59e1267ab6e
+      ROLES: "BUYER,SELLER"
+      PAYMENT_ACCOUNT_DETAILS: "09-34-21 46324354"
+      COLOR: 49200e
+    ports:
+     - 8085:8085
+
+  Farouk:
+    container_name: Farouk
+    image: hederahashgraph/hcs-sxc-java-settlement-demo:latest
+    restart: on-failure
+    environment:
+      APP_ID: 5
+      PUBKEY: 302a300506032b6570032100bcabbb31c4c6418ea323e02dd46c060f82936141b5d2d2a1da89e59e1267ab6f
+      ROLES: "ARBITER,AUDITOR"
+      PAYMENT_ACCOUNT_DETAILS: "10-69-43 86535765"
+      COLOR: 2c0f4f
+    ports:
+      - 8086:8086
+
+  Grace:
+    container_name: Grace
+    image: hederahashgraph/hcs-sxc-java-settlement-demo:latest
+    restart: on-failure
+    environment:
+      APP_ID: 6
+      PUBKEY: 302a300506032b6570032100bcabbb31c4c6418ea323e02dd46c060f82936141b5d2d2a1da89e59e1267ab7a
+      ROLES: "DEMO"
+      PAYMENT_ACCOUNT_DETAILS: "10-32-52 55424452"
+      COLOR: 4f0500
+    ports:
+      - 8087:8087
+=======
```

Comment:
Merge gone wrong ?

Reply:
I had disabled the examples. merging bits only until things work.